src/ng2-modal.ts

```diff
@@ -1,7 +1,7 @@
 import {Component, Input, Output, EventEmitter, ElementRef, ViewChild} from "@angular/core";
```

Comment:
in latest push I renamed ng2-modal to `index.ts`, and right now users will import like this: `import {Modal} from "ng2-modal";`